config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
```

Comment:
Probably a good idea to move this configuration to `Rails.application.secrets`.

Reply:
I think stylistically just replacing the `ENV['STRIPE_PUBLISHABLE_KEY']` with a ref something like `Rails.application.secrets.stripe_publishable_key` but the config makes sense in the initializer, I think.

Also do you mind if we standardise hashes to use the more terse `key: 'val'` than :key => 'val' form. I find writing the rocket takes more time but I don't mind if you feel strongly about it. Would prefer we stick to one or the other though?

Reply:
Ok, cool! Yes, let's standardise on `key: 'val'` notation.